src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs

```diff
@@ -24,7 +24,7 @@ internal partial class InstallDialog : DialogWindow
 
         public InstallDialog(IDependencies dependencies, ILibraryCommandService libraryCommandService, string configFileName, string fullPath, string rootFolder)
         {
-            InstallationFolder.DestinationFolder = fullPath.Replace(rootFolder, "").Replace('\\', '/');
+            InstallationFolder.DestinationFolder = fullPath.Replace(rootFolder, "").TrimStart(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar).Replace('\\', '/');
```

Comment:
>fullPath.Replace(rootFolder, "").TrimStart(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar).Replace('\\', '/') [](start = 51, length = 122)

nit: Break this into multiple lines to make it easier to read.

Reply:
Will do

---
In reply to: [202842892](https://github.com/aspnet/LibraryManager/pull/316#discussion_r202842892) [](ancestors = 202842892)